Dockerfile

```diff
@@ -44,34 +118,86 @@ RUN AVAILABLE_TERRAFORM_VERSIONS="1.0.11 1.1.9 1.2.9 ${DEFAULT_TERRAFORM_VERSION
-        curl -LOs https://github.com/open-policy-agent/conftest/releases/download/v${VERSION}/conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        curl -LOs https://github.com/open-policy-agent/conftest/releases/download/v${VERSION}/checksums.txt && \
-        sed -n "/conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz/p" checksums.txt | sha256sum -c && \
-        mkdir -p /usr/local/bin/cft/versions/${VERSION} && \
-        tar -C /usr/local/bin/cft/versions/${VERSION} -xzf conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        ln -s /usr/local/bin/cft/versions/${VERSION}/conftest /usr/local/bin/conftest${VERSION} && \
-        rm conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        rm checksums.txt; \
-    done
+# Stage 2 - Alpine
+# Creating the individual distro builds using targets
+FROM alpine:${ALPINE_TAG} AS alpine
 
-RUN ln -s /usr/local/bin/cft/versions/${DEFAULT_CONFTEST_VERSION}/conftest /usr/local/bin/conftest
+# atlantis user for gosu and OpenShift compatibility
+RUN addgroup atlantis && \
+    adduser -S -G atlantis atlantis && \
+    adduser atlantis root && \
+    chown atlantis:root /home/atlantis/ && \
+    chmod g=u /home/atlantis/ && \
+    chmod g=u /etc/passwd
 
 # copy binary
 COPY --from=builder /app/atlantis /usr/local/bin/atlantis
+# copy terraform
+# TODO: figure out better way to copy versions over, maybe just track one version
+COPY --from=deps /usr/local/bin/terraform /usr/local/bin/terraform
+COPY --from=deps /usr/local/bin/terraform1.2.9 /usr/local/bin/terraform1.2.9
+COPY --from=deps /usr/local/bin/terraform1.1.9 /usr/local/bin/terraform1.1.9
+COPY --from=deps /usr/local/bin/terraform1.0.11 /usr/local/bin/terraform1.0.11
+# copy deps
+COPY --from=deps /usr/local/bin/conftest /usr/local/bin/conftest
+COPY --from=deps /bin/gosu /bin/gosu
+COPY --from=deps /usr/bin/git-lfs /usr/bin/git-lfs
+# copy docker entrypoint
+COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+
+# Install packages needed for running Atlantis.
+# We place this last as it will bust less docker layer caches when packages update
+RUN apk add --no-cache \
+        ca-certificates~=20220614 \
+        curl~=7.87 \
+        git~=2.38 \
+        unzip~=6.0 \
+        bash~=5.2 \
+        openssh~=9.1_p1 \
+        libcap~=2.66 \
+        dumb-init~=1.2 \
+        gcompat~=1.1
 
+ENTRYPOINT ["docker-entrypoint.sh"]
+CMD ["server"]
+
+# Stage 2 - Debian
+FROM debian:${DEBIAN_TAG} AS debian
+
+# copy binary
+COPY --from=builder /app/atlantis /usr/local/bin/atlantis
+# copy terraform
+# TODO: figure out better way to copy versions over, maybe just track one version
+COPY --from=deps /usr/local/bin/terraform /usr/local/bin/terraform
+COPY --from=deps /usr/local/bin/terraform1.2.9 /usr/local/bin/terraform1.2.9
+COPY --from=deps /usr/local/bin/terraform1.1.9 /usr/local/bin/terraform1.1.9
+COPY --from=deps /usr/local/bin/terraform1.0.11 /usr/local/bin/terraform1.0.11
```

Comment:
I don't know if we want to do this since for each new version, we'd have to update the version in 2 different places.

Reply:
I had reservations about this as well, but it keeps the images cleaner. Ideally, we would only track one version as the latest instead of trying to version multiples. I'm open to suggestions here since we currently manually track TF versions in the for-loop curl logic anyways.

Reply:
One idea is maybe passing in the TF versions as an ARG instead of hard coding and then using a wildcard on the COPY command. (I think wildcard is an accepted syntax, maybe?)

Reply:
If this individual `COPY` system is added, then we should have it setup so `renovatebot` will be able to continue to auto update these. I don't think the current PR addresses this yet.